produc_demo.rb

```diff
@@ -1,12 +1,8 @@
 require_relative 'product.rb'
+require_relative 'vendor.rb'
 
-puts "Create few objects of product class"
+vendor1 = Vendor.new("GoodFood", [Product.new("milk", "15.65"), Product.new("apple", "7.44"), Product.new("pineapple", "14.55")])
+vendor2 = Vendor.new("SportLife", [Product.new("ball", "26"), Product.new("skies", "260"), Product.new("skates", "85")])
```

Comment:
Let's move the created products in separate variable/variables.
Also, let's use single quotes ' instead of " when it is not needed.
As you can see all your prices are numbers, so there is no need to have them as strings.